app/assets/stylesheets/edit_activity.scss

```diff
@@ -0,0 +1,21 @@
+.edit-activity {
+	height: 100vh; width: 100vw;
+
+	/* .activities */
+	.activities {
+		position: relative; top: 10%;
+		text-align: center;
+		h2 { font-size: 5em; }
+	}
+	/* .activities */
+
+	.quote-banner {
+		h3 {font-size: 1.8em;}
+	}
+
+	.todays_time {
+		
```

Comment:
Please work on styling of code. Indentation, adding necessary linebreaks and removing unnecessary line breaks makes the code  alot more readable.